### ISSUE TITLE
what's the difference between scatter_softmax and softmax in pyg lib

### ISSUE THREAD

**Issue by NIRVANALAN** [rusty1s/pytorch_geometric#877]
## ❓ Questions & Help

Hi, I noticed you have a softmax in torch_geometric.utils and softmax functions in torch_scatter lib. Are they the same? Because I need to use scatter_log_softmax now and I wonder if it equals calling 
`torch.log(softmax(src, index)+eps)`

**Comment by rusty1s** [comment]
We will deprecate the softmax version of PyG soon! So I recommend going with the scatter versions.

**Comment by NIRVANALAN** [comment]
thanks for your reply, but how can i transfer from PyG version to scatter version?
![image](https://user-images.githubusercontent.com/23359323/71504512-ad017680-28b4-11ea-9b18-bb9c3e0aa9cb.png)
I want to do softmax in dim=0 and this works find in original softmax

**Comment by rusty1s** [comment]
Oh, you are right. The scatter version does not yet support broadcasting. I will fix this. So please use the PyG version for now :)

**Comment by NIRVANALAN** [comment]
Thanks for your help!
